.github/workflows/ci.yml

```diff
@@ -17,9 +17,6 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
 
-      - name: Install npm@7
```

Comment:
Since this library doesn't have any peerDeps we don't really need to run the install in CI with any specific version of NPM; whatever's available with the version of Node that we're using is fine.